### PR TITLE
Added Zoot module.

### PR DESCRIPTION
**Pull request by ghost**
* Also added a check for permanent potions.

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleZoot.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    val noAir by boolean("NoAir", false)
+
+    val repeatable = repeatable {
+        if (!noAir && player.isOnGround) {
+            if (badEffects) {
+                val effect = player.activeStatusEffects.maxByOrNull { it.value.duration }
+
+                if (effect != null) {
+                    if(effect.value.isPermanent) {
+                        player.statusEffects.remove(effect.value)
```
Comment (1zun4secondary) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleZoot.kt:21]: Are you crazy?

Comment (1zun4secondary) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleZoot.kt:21]: Don't just remove the potion effects from the client side.

Comment (1zun4secondary) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleZoot.kt:21]: Please fix this.

Comment (ghost) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleZoot.kt:21]: How should I remove potions server side ? Mixins again?

Comment (ghost) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleZoot.kt:21]: Found a packet for that.

**Pull request merged**